Code/C++/reverse_string.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+PROBLEM STATEMENT:
+Given a string, print its reverse.
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+//Driver function
+int main(){
+	string str;
+	getline(cin, str);	//Taking input using getline() to include whitespaces
+	reverse(str.begin(), str.end());	//In-built function to reverse a string

```

Comment:
Write the code without include stl,  you may tell about this stl in the description or the comments in between the code.